openmp/runtime/src/include/ompx.h.var

```diff
@@ -81,6 +83,10 @@ _TGT_KERNEL_LANGUAGE_HOST_IMPL_SYNC_C(void, sync_block_divergent, int Ordering,
 #undef _TGT_KERNEL_LANGUAGE_HOST_IMPL_SYNC_C
 ///}
 
+static INLINE uint64_t ompx_ballot_sync(uint64_t mask, int pred) {
+  __builtin_trap();
+}
```

Comment:
What's this for? If it's a host fallback I would just return `mask & pred`.

Reply:
We don't want application running on host using these functions to run, especially soon I'll add shuffle.

Reply:
Why define it on the host at all then?

Reply:
This is more like a temporary solution before we force the front end to stop emitting host fallback when kernel lang mode is turned on automatically; otherwise undefined symbol link error would appear.